common:
    lid_topic:  "/os1_cloud_node1/points"

preprocess:
    lidar_type: 3                # 1 for Livox serials LiDAR, 2 for Velodyne LiDAR, 3 for ouster LiDAR, 
    scan_line: 16
    scan_rate: 10                # only need to be set for velodyne, unit: Hz,
    timestamp_unit: 3                 # 0-second, 1-milisecond, 2-microsecond, 3-nanosecond.
    Horizon_SCAN: 1024           # lidar horizontal resolution (Velodyne:1800, Ouster:512,1024,2048)
    blind: 2
    has_ring: true

normal:
    ringfals_en: true
    compute_table: false
    ring_table_dir: "/config/viral"       # lookup table path, read or write
